pcweb/pages/customers/stories/bayesline.md

```diff
@@ -0,0 +1,106 @@
+---
+company: Bayesline
+description: "How Bayesline builds 4x faster using Reflex instead of Plotly Dash"
+domain: "https://bayesline.com/"
+founded: "New York, 2024"
+investors: "Y Combinator",
+meta: [
+  {
+    "name": "keywords",
+    "content": "
+      plotly dash,
+      python and react,
+      python and react js,
+      python react,
+      python web app,
+      python web app framework,
+      python web apps,
+      react js python,
+      react js with python,
+      react python,
+      react python websocket,
+      react with python,
+      web app framework python,
+      web app python,
+    "
+  }
+]
+---
+
+Why Bayesline chose Reflex instead of Plotly Dash
+
+## TLDR;
+
+Sebastian and his small team use Reflex to keep the full flexibility of a traditional ReactJS app **WITHOUT** needing to:
+
+* Learn frontend: React (JavaScript), NodeJS, TailwindCSS
+
+```md quote Misha
+Reflex definitely saved us from needing to hire a frontend engineer.
+```
+
+* Write boilerplate to stitch together their frontend and backend:
+
+```md quote Sebastian
+50% less code than Dash
+```
+
+* Maintain expensive, fragile, and inevitably slow Dash apps; using Reflex instead of Plotly Dash
+
+```md quote Misha
+Using Reflex instead of Plotly Dash was like the difference between organized Legos and a plate of spaghetti
+```
+
+## What Baylesline is building
+
+Meet Bayesline, the Y-combinator-backed GPU-powered financial analytics suite for institutional investors.
+Bayesline deploys customized equity factor risk (ML) models in the cloud, which previously could take weeks or months for investors to build.
+Sebastian Janisch, the Bayesline co-founder, led the technical team and worked previously at Bloomberg and BlackRock, where he was a Director in their Financial Modeling Group.
+Here is a blog that further shows what Bayesline does in further detail [https://bayesline.com/blog/building-risk-models](https://bayesline.com/blog/building-risk-models) (The images and demos are all Reflex apps).
+
+## How Bayesline builds 4x faster using Reflex instead of Plotly Dash
```

Comment:
This is not correct, they build 4x faster than React not dash, so we cannot say this, we want to mention why we are a better comparison to dash as well